### DISCUSSION TITLE
Inquiry of CU/DU server for split 7.2

### DISCUSSION THREAD

**Discussion by MohammadYAmmar** [srsran/srsRAN_Project#631] [General Help]
Hello srsRAN users,

Can you please provide me with the recommended specifications for a CU/DU server to use split 7.2 with srsRAN.

Any help would be appreciated and I hope this helps others too; Because it is not found in the documentation other than the experiment device, which is a regular specs of computer

**Comment by andrepuschmann** [comment]
Unfortunately, there is no one-size-fits-all answer. The recommended specs for a system very much depend on the intended configuration, such as #cells, bandwidth, MIMO layers, etc. 

The spec we've given in the docs (e.g. Ryzen 5700G) is just an example system we built some time ago. But just to get you an idea, with this system you can easily run a single cell 100MHz 2T2R and achieve full rate of 700-800 Mbit/s with a COTS UE with the entire CU/DU and the 5GC on a single system. Without even tuning much.

Hope that helps.

**Reply by MohammadYAmmar** [reply]
Thank you so much, What is the procedure to interface single unit split 7.2 Bentel 550 in-door radio with Ryzen 5700G CPU, PC. 
Do we need to get installed eCPRI interface card in the PC, we need a PTP switch or built-in card to support eCPRI interface?

**Reply by pgawlowicz** [reply]
@MohammadYAmmar please check this tutorial: https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/index.html
